openlibrary/core/search.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import absolute_import
```

Comment:
This is a _super_ confusing name for this module. I expected this to deal with, like, searching? Like the search page? Whereas this has nothing to do with that :P Maybe `ia_query`?

Reply:
This has confused me recently in another unrelated bit of code chasing, `search` implies OL search functionality -- interacting with IA via its elastic search API is different and should be described better.